samples/cpp/common/format_reader/CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ file (GLOB LIBRARY_PUBLIC_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.h)
 source_group("src" FILES ${LIBRARY_SRC} ${LIBRARY_HEADERS})
 source_group("include" FILES ${LIBRARY_PUBLIC_HEADERS})
 
+if(AARCH64)
+    if(APPLE)
+        ov_add_compiler_flags(-Wno-shorten-64-to-32)
+    endif()
+endif()
```

Comment:
`ov_add_compiler_flags` can be unavailable in this place. It's part of samples code:
- when samples are built as a part of whole OpenVINO build - this function is here
- when samples are part of OpenVINO release package, this function is not available, because it exists only in OpenVINO source tree (part of developer cmake scripts).

So, we have to deal with standard cmake variables CMAKE_CXX_FLAGS, CMAKE_C_FLAGS. Maybe we can just fix the warnings in this module?